tools/orca/src/main.rs

```diff
@@ -205,74 +206,88 @@ async fn main() -> ExitCode {
                     return ExitCode::FAILURE;
                 }
             };
-
+            // here we need to create one less worker compared to the desired amount since we later call `spawn_blocking`, which consumes
+            // an extra thread all on its own
+            let runtime = build_tokio_runtime(state.thread_count.map(|t| t - 1));
```

Comment:
No, you don't need to do this. spawn blocking is a separate thread pool. Just set the thread count to the number of threads.

Reply:
But my fear was that since that uses a separate thread pool, if we have 20 workers and 1 spawn blocker, won't this use 21 threads?

Reply:
And why is that a problem? That's what spawnblocking does. If you want to maek it 19 workers, set it to 19.